packages/@webex/plugin-cc/src/services/core/WebSocket/WebSocketManager.ts

```diff
@@ -59,6 +61,10 @@ export class WebSocketManager extends EventTarget {
     }
   }
 
+  handleConnectionLost(event: ConnectionLostDetails) {
+    this.isConnectionLost = event.isConnectionLost;
```

Comment:
Had forgotten to include this, it is needed here. This gets called when the connection is lost (event will be sent to cc which will call this).
Agent desktop was using a public signal variable, we have used events.